src/globus_cli/commands/group/member/add.py

```diff
@@ -0,0 +1,45 @@
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import IdentityType, ParsedIdentity, command
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+from globus_cli.types import FIELD_LIST_T
+
+
+def _format_added_user_id(x: dict):
+    return x["add"][0]["identity_id"]
+
+
+def _format_added_user_username(x: dict):
+    return x["add"][0]["username"]
+
+
+def _format_group_id(x: dict):
+    return x["add"][0]["group_id"]
```

Comment:
Unfortunately, there's an issue with unpacking the response like this.
Groups responds with an empty array if the user is already in the group, i.e. `{"add": []}`.

We're going to need to make some change to handle it. I think these helpers can stay unmodified, and I'll provide a suggestion down below, where `formatted_print` is called.

Reply:
Thanks, I was thinking this might have been flaky and this would need to be discussed. Thanks for such detailed suggestions.